modules/social_features/social_topic/social_topic.module

```diff
@@ -39,11 +40,12 @@ function social_topic_preprocess_node(array &$variables) {
     $topic_type = $node->get('field_topic_type');
     $topic_type_entities = $topic_type->referencedEntities();
     if (count($topic_type_entities) === 1) {
+      $curr_langcode = \Drupal::languageManager()->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
       foreach ($topic_type_entities as $topic) {
         $variables['metadata'] = t('in @topic', ['@topic' => $topic->link()]);
```

Comment:
@slowflyer It looks like this may also be applicable for the metatdata variable?

Reply:
@Kingdutch  no metadata shows up correctly translated

Reply:
@slowflyer I don't quite understand why `$topic->link()` does show up translated (it creates a link of the label and actually calls `$this->label()`) but the `$topic->label()` call itself does not work.

Unless I misunderstand in this case and `$variables['metadata']` doesn't need to be translated.

Reply:
@Kingdutch maybe you are right ... I do not translate the link ... that may be the reason why everything looks fine for me. I will add the logic for $topic-link() as well